com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/flowable/FlowableFacade.java

```diff
@@ -35,6 +36,7 @@
 
     private static final int DEFAULT_JOB_RETRIES = 0;
     private static final int DEFAULT_ABORT_TIMEOUT_MS = 30000;
+    private static final String VAR_USER = "user";
```

Comment:
FlowableFacade is a facade and should be abstract enough. It should not be aware of 'our internal variables' that we use. Read the comment below for more clarification.